src/System.Private.Windows.Core/src/Windows/Win32/Graphics/Gdi/RegionScope.cs

```diff
@@ -15,7 +16,7 @@ namespace Windows.Win32.Graphics.Gdi;
 ///  </para>
 /// </remarks>
 #if DEBUG
-internal class RegionScope : DisposalTracking.Tracker, IDisposable
+internal unsafe class RegionScope : DisposalTracking.Tracker, IDisposable
 #else
 internal ref struct RegionScope
```

Comment:
Think we may also need unsafe here
```suggestion
internal unsafe ref struct RegionScope
```